packages/amplify-frontend-ios/index.js

```diff
@@ -60,15 +60,17 @@ async function executeAmplifyCommand(context) {
 }
 
 async function handleAmplifyEvent(context, args) {
-  switch (args.event) {
-    case 'PostInit':
-      await onPostInit.run(context, args);
-      break;
-    case 'PostCodegenModels':
-      await onPostCodegenModels.run(context, args);
-      break;
-    default:
-      break;
+  const postEvents = new Set(['PostInit', 'PostCodegenModels', 'PostPull']);
```

Comment:
Can you move this out of the function so that the `Set` only needs to be created once.

Reply:
Sure, done